api-server/src/main/java/io/enmasse/api/server/ApiServer.java

```diff
@@ -107,9 +107,12 @@ public void start(Future<Void> startPromise) throws Exception {
         Metrics metrics = new Metrics();
 
         HTTPHealthServer httpHealthServer = new HTTPHealthServer(options.getVersion(), metrics);
-        HTTPServer httpServer = new HTTPServer(addressSpaceApi, schemaProvider, authApi, userApi, options, clientCa, requestHeaderClientCa, clock, authenticationServiceRegistry, apiHeaderConfig, metrics);
 
-        vertx.deployVerticle(httpServer, new DeploymentOptions().setWorker(true), result -> {
+        ResteasyDeploymentFactory resteasyDeploymentFactory = new ResteasyDeploymentFactory(addressSpaceApi, schemaProvider, authApi, userApi, clock, authenticationServiceRegistry, apiHeaderConfig, metrics, options.isEnableRbac());
+        String finalRequestHeaderClientCa = requestHeaderClientCa;
+        String finalClientCa = clientCa;
+        vertx.deployVerticle(() -> new HTTPServer(options, resteasyDeploymentFactory, finalClientCa, finalRequestHeaderClientCa),
+                new DeploymentOptions().setWorker(true).setInstances(options.getNumWorkerThreads()), result -> {
```

Comment:
Maybe log options.getNumWorkerThreads() at info level too.

Reply:
All options (including this) are printed at info level during startup